website/content/docs/job-specification/disk_throttle.mdx

```diff
@@ -0,0 +1,113 @@
+---
+layout: docs
+page_title: disk_throttles block in the job specification
+description: |-
+  Configure disk I/O throttling limits in the `disk_throttles` block of the Nomad job specification.
+---
+
+
+# `disk_throttle` block in the job specification
+
+
+<Placement groups={['job', 'group', 'task', 'resources', 'disk_throttle']} />
+
+The disk_throttle block is used to set limits on the disk I/O (Input/Output) a task can perform on a specific block device. 
+This block helps mitigate the "noisy neighbor" problem, where a single task consuming excessive disk bandwidth can negatively impact other tasks running on the same node.
+
+When a disk_throttle block is added, Nomad will limit the task's I/O throughput in bytes per second (BPS) or I/O operations per second (IOPS). 
```

Comment:
```suggestion
When you add a `disk_throttle` block, Nomad limits the task's I/O throughput in bytes per second (BPS) or I/O operations per second (IOPS). 
```

- [Use active voice](https://github.com/hashicorp/web-unified-docs/blob/main/docs/style-guide/top-12.md#write-in-active-voice)
- [Use present tense](https://github.com/hashicorp/web-unified-docs/blob/main/docs/style-guide/top-12.md#use-simple-present-tense-to-describe-immediate-actions)